cockatrice/src/dlg_filter_games.cpp

```diff
@@ -63,12 +63,12 @@ DlgFilterGames::DlgFilterGames(const QMap<int, QString> &_allGameTypes,
     generalGroupBox = new QGroupBox(tr("General"));
     generalGroupBox->setLayout(generalGrid);
 
-    QVBoxLayout *gameTypeFilterLayout = new QVBoxLayout;
+    auto *gameTypeFilterLayout = new QVBoxLayout;
     QMapIterator<int, QString> gameTypesIterator(allGameTypes);
     while (gameTypesIterator.hasNext()) {
```

Comment:
while you're at it with the refactoring, can you format this too?

Reply:
Sure, what formatting would you like changed? Sorry if missing something obvious

Reply:
I prefer the `for` syntax instead of java style, but we don't even call attention to this in the CONTRIBUTING.md so just leave it as is.